lib/index.js

```diff
@@ -73,6 +87,7 @@ function init (options = {}) {
     app.get(oauth2Settings.path, auth.express.authenticate(name, oauth2Settings));
     app.get(
```

Comment:
Would it be safe to just add the `cookieParser` middleware here?

Reply:
Yes it would, however, social linking would not work if cookies aren't enabled by the developer for `@feathersjs/authentication-jwt`.

Reply:
I think that should be fine, since you need to do that when using the client with oAuth2 anyway (the generator enables cookies automatically in that case already).